packages/openapi-ts/test/__snapshots__/test/generated/v3_models/models.ts.snap

```diff
@@ -722,6 +722,10 @@ export type ModelWithReadOnlyAndWriteOnly = {
     baz: string;
 };
 
+export type ModelWithConstantSizeArray = [number];
+
+export type ModelWithAnyOfConstantSizeArray = [number, string];
```

Comment:
Shouldn't this produce

```suggestion
export type ModelWithAnyOfConstantSizeArray = [number | string, number | string, number | string];
```

since minItems and maxItems is 3 and they're oneOf?

Reply:
Will be fixed in other PR